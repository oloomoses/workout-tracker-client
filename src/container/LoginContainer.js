import React from 'react';
import Login from '../components/Login';

const LoginContainer = () => (
  <div>
    <Login />
  </div>
);

export default LoginContainer;
