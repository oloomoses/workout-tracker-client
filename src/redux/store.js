import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import userRegistrationReducer from './user/userRegisterReducer';

const store = createStore(userRegistrationReducer, applyMiddleware(thunk));

export default store;
