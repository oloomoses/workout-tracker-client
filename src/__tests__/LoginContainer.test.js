import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import LoginContainer from '../container/LoginContainer';
import store from '../redux/store';

describe('LoginContainer component', () => {
  it('Matches the snapshot', () => {
    const tree = renderer.create(<Provider store={store}><LoginContainer /></Provider>).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
