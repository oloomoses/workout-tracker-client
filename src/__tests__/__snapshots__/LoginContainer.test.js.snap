// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer component Matches the snapshot 1`] = `[Function]`;
